sections/futures/Trade/TradePanelHeader.tsx

```diff
@@ -31,19 +26,9 @@ export default function TradePanelHeader({ accountType, onManageBalance }: Props
 	const dispatch = useAppDispatch();
 	const theme = useTheme();
 
-	const { openConnectModal } = useConnectModal();
-	const wallet = useAppSelector(selectWallet);
 	const position = useAppSelector(selectPosition);
 	const positionStatus = useAppSelector(selectPositionStatus);
 	const balance = position ? position.remainingMargin : null;
-	const openModal = useAppSelector(selectOpenModal);
-	if (!wallet) {
```

Comment:
We can leave this as it was now that TransferIsolatedMarginModal is not below it any more.

Reply:
Reverted. Thanks.